include/st_panel.h

```diff
@@ -0,0 +1,12 @@
+
+#if defined HAVE_NCURSESW_PANEL_H
+#include <ncursesw/panel.h>
+#elif defined HAVE_NCURSES_PANEL_H
+#include <ncurses/panel.h>
+#elif defined HAVE_PANEL_H
+#include <panel.h>
+#else
+/* fallback */
```

Comment:
@okbob @tysenmoore-xse 
If configure did not found the panel header then isn't it wrong to still include it?
Note: the old existing "error" in the original code from @okbob in st_menu.c was just changed:

```diff
 #if defined HAVE_NCURSESW_PANEL_H
 #include <ncursesw/panel.h>
 #elif defined HAVE_NCURSES_PANEL_H
 #include <ncurses/panel.h>
 #elif defined HAVE_PANEL_H
 #include <panel.h>
 #else
 /* fallback */
-#include <ncurses/panel.h>
+#include <panel.h>
 #endif
```

I suggest to place an `#error missing panel header` there, similar for st_curses.h.

Reply:
If I remember well, I did it when I worked on Solaris support. The correct solution should be fixing m4_ax_with_curses_extra.m4 script to detect this situation. But I am not able to do it :(. Importunately it is hell to work with ncurses headers. RH, Debian, FreeBSD, Solaris has different rules for placing of these files. Current state of pspg or ncurses-st-menu is little but messy, but it is almost work.

Reply:
Hm, you could still pass an option to configure: `CPPFLAGS="-DHAVE_NCURSES_PANEL_H` or similar in the case of "something does not work"; and if I did spot this right when I had a quick look in configure.ac: it would error and stop the building if it doesn't find the library/header.
@tysenmoore-xse I'd argue to at least don't change the "bad" fallback...